pkg/virt-handler/virtwrap/api/converter_test.go

```diff
@@ -330,7 +330,7 @@ var _ = Describe("Converter", func() {
       <source protocol="iscsi" name="iqn.2013-07.com.example:iscsi-nopool/2">
         <host name="example.com" port="3260"></host>
       </source>
-      <target bus="virtio" dev="vda"></target>
+      <target bus="ide" dev="hdg"></target>
```

Comment:
I want to add more tests in a future upload - or even in a future PR if the reviewers prefer so.